packages/langchain/src/openai/util.test.ts

```diff
@@ -39,6 +42,18 @@ describe('Mapping Functions', () => {
       new SystemMessage('System Test Content'),
       new AIMessage('AI Test Content')
     ];
+    const addNumbersSchema = z
+      .object({
+        a: z.number().describe('The first number to be added.'),
+        b: z.number().describe('The second number to be added.')
+      })
+      .strict();
+
+    const myTool = tool(() => {}, {
+      name: 'test',
+      description: 'Some description',
+      schema: addNumbersSchema
+    });
```

Comment:
I suggest that we stick to LangChain style in our LangChain client using `tool()` function to create tools.

https://js.langchain.com/docs/how_to/custom_tools/#tool-function